ibu-imager/postpivot/postpivot.go

```diff
@@ -21,51 +21,53 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 
-	"github.com/openshift-kni/lifecycle-agent/ibu-imager/clusterinfo"
+	clusterconfig_api "github.com/openshift-kni/lifecycle-agent/api/seedreconfig"
 	"github.com/openshift-kni/lifecycle-agent/ibu-imager/ops"
+	"github.com/openshift-kni/lifecycle-agent/ibu-imager/seedclusterinfo"
 	"github.com/openshift-kni/lifecycle-agent/internal/common"
 	"github.com/openshift-kni/lifecycle-agent/internal/recert"
 	"github.com/openshift-kni/lifecycle-agent/utils"
 )
 
 type PostPivot struct {
-	scheme               *runtime.Scheme
-	log                  *logrus.Logger
-	ops                  ops.Ops
-	recertContainerImage string
-	authFile             string
-	workingDir           string
-	kubeconfig           string
+	scheme     *runtime.Scheme
+	log        *logrus.Logger
+	ops        ops.Ops
+	authFile   string
+	workingDir string
+	kubeconfig string
 }
 
-func NewPostPivot(scheme *runtime.Scheme, log *logrus.Logger, ops ops.Ops,
-	recertContainerImage, authFile, workingDir, kubeconfig string) *PostPivot {
+func NewPostPivot(scheme *runtime.Scheme, log *logrus.Logger, ops ops.Ops, authFile, workingDir, kubeconfig string) *PostPivot {
 	return &PostPivot{
-		scheme:               scheme,
-		log:                  log,
-		ops:                  ops,
-		authFile:             authFile,
-		recertContainerImage: recertContainerImage,
-		workingDir:           workingDir,
-		kubeconfig:           kubeconfig,
+		scheme:     scheme,
+		log:        log,
+		ops:        ops,
+		authFile:   authFile,
```

Comment:
Where are we getting the RecertImage now?

Reply:
It's part of `SeedClusterInfo`